apps/homework/templates/homework/forms/grade_homework.html

```diff
@@ -81,6 +91,9 @@ <h4>Questions</h4>
                                    target="_blank" rel="noopener noreferrer"
                                 >{{ submission.publication_url }}</a></li>{% endif %}
                     </ol>
+                    {% if not definition.ask_project_url and not definition.ask_publication_url and not definition.ask_method_name and not definition.ask_method_description %}
```

Comment:
this seems kinda grody, could this become a model property or some such?

Reply:
Very grody. Can do. Thanks!

Reply:
✅